2.0/conn/setup_db.py

```
@@ -0,0 +1,36 @@
+
+def create_db_tables():
+    '''
+    Enables pgvector (extension must be installed) and creates the model classes found in Base.
+    '''
+
+    # TODO: Proper indexing for embedding tables
+    
+    with engine.connect() as conn:
+        conn.execute(text("CREATE EXTENSION IF NOT EXISTS vector"))
+        conn.commit()
+    logging.info(f'Enabled pgvector extension in DB (if not already).')
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)